### ISSUE TITLE
Winfita (T532) Treadmill Incline Levels Table for Calculations, etc.

### ISSUE THREAD

**Issue by laurenweinstein1** [cagnulein/qdomyos-zwift#1244]
The incline levels of the Winfita (T532) treadmill are highly nonlinear, making calculations of elevation gain and other related factors problematic. Dealing with this probably needs a lookup table conversion in QZ, ideally modifiable by the user to account for variations between units, which may be considerable and would need to account for any user modifications of their devices (e.g., in my case the rear of the treadmill is raised ~3.5cm since the lowest incline level of this treadmill is surprisingly high).

Attached is the table of best incline percentage measurements I could get with the equipment at hand directly from the treadmill, again keeping in mind that the rear on the one I'm working with is raised ~3.5cm. 

The treadmill has 16 incline levels from 0 to 15. The first column of measurements are the incline percentage measurements captured for each level on the way up, and the second column are the measurements for each level on the way down (reversed top to bottom for easier comparison). Obviously there is variation between these sets of measurements.

Also attached is the table of "official" incline measurements per level for this model of treadmill, which didn't seem very accurate when the first couple of levels were checked on this particular unit before the rear was raised.

The ideal solution would seem to be a table pre-populated with default values for each incline level, but that also permits the user to replace these values with their own measured values for their own situation and unit.

Thanks!

MEASURED VALUES:

[chartAA.txt](https://github.com/cagnulein/qdomyos-zwift/files/10611496/chartAA.txt)

![Selection_651](https://user-images.githubusercontent.com/11930128/216830914-65f4ff36-51d1-4d59-a921-6263a80e30a6.png)

OFFICIAL VALUES:

![Selection_650](https://user-images.githubusercontent.com/11930128/216830818-af99d1e1-d064-47a1-a3d4-63451cc4e9dd.png)

**Comment by cagnulein** [comment]
yes my idea is to create a general 30 rows table (or more) one for every 0.5 step so the user can override the incline values. do you agree?

**Comment by laurenweinstein1** [comment]
Yeah, that would definitely do the trick. Only side issue would be whether to put it in percentages or degrees or both.

**Comment by cagnulein** [comment]
percentage is better from my code side

**Comment by laurenweinstein1** [comment]
That would be just fine.

**Comment by cagnulein** [comment]
started the development, i "just" need to link the settings to the code
